plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -618,10 +647,11 @@ global proc mayaUsdMenu_markingMenuCallback( string $obj )
 {
     if (!hasPrimUpdater())
         return;
-
-    if (isPulledUsdObject($obj)) {
-        string $pushback = `menuItem -label "Merge Maya Edits to USD" -insertAfter "" -image "merge_to_USD.png" -command ("mayaUsdMenu_pushBackToUSD " + $obj)`;
-        string $pushclear = `menuItem -label "Discard Maya Edits" -insertAfter $pushback -image "discard_edits.png" -command ("mayaUsdDiscardEdits " + $obj)`;
+    
+    string $pulledObject = getPulledUsdObject($obj);
+    if ($pulledObject != "") {
+        string $pushback = `menuItem -label "Merge Maya Edits to USD" -insertAfter "" -image "merge_to_USD.png" -command ("mayaUsdMenu_pushBackToUSD " + $pulledObject)`;
+        string $pushclear = `menuItem -label "Discard Maya Edits" -insertAfter $pushback -image "discard_edits.png" -command ("mayaUsdDiscardEdits " + $pulledObject)`;
```

Comment:
I realize you didn't add these strings, but these two (and the two below) should be in the string resource.

Reply:
Will do it in a separate PR once this one is merged.